exodus_lambda/functions/origin_response/origin_response.py

```diff
@@ -24,6 +24,14 @@ def handler(self, event, context):
         request = event["Records"][0]["cf"]["request"]
         response = event["Records"][0]["cf"]["response"]
 
+        if "headers" in request and "want-digest" in request["headers"]:
+            response["headers"]["digest"] = [
+                {
+                    "key": "Digest",
+                    "value": f"sha-256={request['uri']}".replace("/", "", 1),
```

Comment:
One more thing: after re-reading the spec, I think the correct digest for us to use is actually id-sha-256.  This explicitly means "digest of the content without encoding", which is always what we've stored in S3, so it makes sense.

sha-256 instead can be affected by encoding, so that could possibly be wrong if there are certain features enabled on the CDN like transparent compression (Content-Encoding: gzip or similar).

The only change is that sha-256= should be instead id-sha-256=.